samples/apps/bookinfo/cluster_extension.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+#
+# Copyright 2017 Istio Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# Script to run on a VM with VPC connectivity to the cluster, to prepare the DB and details app.
```

Comment:
VPC connectivity is confusing as it is not required.  can we reword it to network connectivity?

Reply:
Yes, I think ratings should run on k8s, to show it can access a DB over TCP. That's 
the main use case.

Details is to show http works - and can run either on the VM or on k8s. Why - simply easiest to install (just ruby dep). I tried the others - any can be run, and we can even have a demo with all components on VMs, but this seems the lighter.

On wording - 'network connectivity' is also pretty confusing.